reference/twig_reference.rst

```diff
@@ -355,6 +355,38 @@ Returns the absolute URL (with scheme and host) for the given route. If
 ``schemeRelative`` is enabled, it'll create a scheme-relative URL. More
 information in :ref:`book-templating-pages`.
 
+absolute_url
+~~~~~~~~~~~~
+
+.. code-block:: jinja
+
+    {{ absolute_url(path) }}
+
+``path``
+    **type**: ``string``
+
+Returns the absolute URL for the given absolute path. This is useful to convert
+an existing path:
+
+.. code-block:: jinja
+
+    {{ absolute_url(asset(path)) }}
+
+relative_path
+~~~~~~~~~~~~~
+
+.. code-block:: jinja
+
+    {{ relative_path(path) }}
+
+``path``
+    **type**: ``string``
+
+Returns a relative path for the given absolute path (based on the current
+request path). For instance, if the current path is
+``/article/news/welcome.html``, the relative path for path
```

Comment:
I would remove the second "path" completely: "the relative path for `/article/image.png` is `../images.png`"